app/navigators/TabNavigator.tsx

```diff
@@ -90,7 +97,7 @@ export function TabNavigator() {
         <Tab.Screen
           key={screen.name}
           name={screen.name}
-          component={screen.component}
+          component={screen.component as ComponentType}
```

Comment:
I'm not so sure about this one.

Reply:
An easier strategy would be to extract this out of an array and write these inline manually like other navigators.

<img width="618" alt="Screenshot 2023-05-09 at 11 02 52 AM" src="https://github.com/infinitered/ChainReactApp2023/assets/37849890/7b8c4afe-74ff-4a91-adb6-8242f3b12c18">